qiskit/tools/visualization/_qcstyle.py

```diff
@@ -70,7 +70,7 @@ def __init__(self):
         self.dpi = 150
         self.margin = [2.0, 0.0, 0.0, 0.3]
         self.cline = 'doublet'
-        self.reverse = False
+        self.reverse_bits = False
```

Comment:
This can actually be removed, it was missed as part of https://github.com/Qiskit/qiskit-terra/pull/1567 which removed support for using `reverse_bits` and `plot_barriers` from the style dict.